traefik/templates/_helpers.tpl

```diff
@@ -213,17 +213,22 @@ The version can comes many sources: appVersion, image.tag, override, marketplace
 
 {{/* Generate/load self-signed certificate for admission webhooks */}}
 {{- define "traefik-hub.webhook_cert" -}}
-{{- $cert := lookup "v1" "Secret" (include "traefik.namespace" .) "hub-agent-cert" -}}
-{{- if $cert -}}
-{{/* reusing value of existing cert */}}
+{{- if $.Values.hub.apimanagement.admission.customWebhookCertificate }}
+Cert: {{ index $.Values.hub.apimanagement.admission.customWebhookCertificate "tls.crt" }}
+Key: {{ index $.Values.hub.apimanagement.admission.customWebhookCertificate "tls.key" }}
+{{- else -}}
+    {{- $cert := lookup "v1" "Secret" (include "traefik.namespace" .) "hub-agent-cert" -}}
```

Comment:
Here it should look for the name in the bugfix below.

Reply:
I opened PR https://github.com/traefik/traefik-helm-chart/pull/1415 to solve this point.